### PR TITLE
BZ-207662-main:adds to important message

### PR DESCRIPTION
**Pull request by JoeAldinger**
BZ 207662

Version(s):
4.8+

Issue:
[BZ 2076662](https://bugzilla.redhat.com/show_bug.cgi?id=2076662#c16)

Link to docs preview:
[Preview](https://joealdinger.github.io/openshift-docs/BZ-207662-main/networking/configuring_ingress_cluster_traffic/configuring-externalip.html) for change to IMPORTANT callout.
4.8.34 [preview](https://joealdinger.github.io/openshift-docs/BZ-2076662-4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-34). These will be the same changes made to 4.9 and 4.10. 

Additional information:
These PRs address the changes across the impacted branches and zstream releases. 
PR to update 4.8.34 #46625
PR to update 4.9.23 #46659 
PR to update 4.10.3 #47303

### CODE REVIEW

**Comment by JoeAldinger** [comment]
PTAL @CFields651 @xltian @tssurya

File: modules/nw-externalip-about.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ If routed correctly, external traffic from the configured external IP address bl
 
 [IMPORTANT]
 ====
-You must ensure that the IP address block you assign terminates at one or more nodes in your cluster.
```
Comment (tssurya) [inline modules/nw-externalip-about.adoc:54]: unrelated to your PR, but this sentence is just confusing `terminates at one of more nodes in your cluster?` what does that mean?

**Review by tssurya — APPROVED** [review]
LGTM to the actual change

File: modules/nw-externalip-about.adoc [hunk 2]
```diff
@@ -51,7 +51,7 @@ If routed correctly, external traffic from the configured external IP address bl
...
+As a cluster Administrator, it is your responsibility to configure routing to externalIPs on both SDN and OVN. You must also ensure that the IP address block you assign terminates at one or more nodes in your cluster.
```
Comment (tssurya) [inline modules/nw-externalip-about.adoc:54]: A slight suggestion; I don't know if you provide links to other repos, but you could add https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/external-ip-and-loadbalancer-ingress.md#guidance-for-administrators OR https://kubernetes.io/docs/concepts/services-networking/service/#external-ips as external link where users can read more.

**Pull request merged**